gcasc/__init__.py

```diff
@@ -83,8 +84,10 @@ def configure(self, target=None):
                     self._configure(configurer)
                 except GcascException as exc:
                     exceptions.handle_gcasc_exception(exc, logger)
+                    sys.exit(1)
```

Comment:
Could you please include that in tests? ;) 

Unfortunately there are no tests for verifying exception handling yet. But imo they should be here: https://github.com/Roche/gitlab-configuration-as-code/blob/master/tests/gcasc_test.py

Reply:
Done! xD finally